client/benches/tps/lib.rs

```diff
@@ -100,7 +100,7 @@ impl Config {
             .sumeragi
             .wsv_mutex_access()
             .clone();
-        let mut blocks = blocks_wsv.blocks().skip(blocks_out_of_measure as usize);
+        let mut blocks = blocks_wsv.all_blocks_by_value().into_iter().skip(blocks_out_of_measure as usize);
```

Comment:
```suggestion
        let mut blocks = blocks_wsv.blocks().into_iter().skip(blocks_out_of_measure as usize);
```

The `by_value` part is implied by `into_iter()`, and the fact that there are `all` blocks is implied by the fact that you apply `filter`, `skip` and other `iterator` methods after.

Reply:
The `by_value` is referring to how destructive the function is to performance. Perhaps the normal reader does not consider it insane to get "*all the blocks by* **value**". How would you name this in such a way that it shouts "really don't use this, it is terrible for performance".